obs-mcp/pkg/prometheus/guardrails.go

```diff
@@ -31,6 +40,8 @@ func DefaultGuardrails() *Guardrails {
 		DisallowExplicitNameLabel: true,
 		RequireLabelMatcher:       true,
 		DisallowBlanketRegex:      true,
+		MaxMetricCardinality:      20000,
+		MaxLabelCardinality:       500,
```

Comment:
Curious about how these were decided? I remember us talking about gauging these from the observed telemetry data in one of the previous calls.

Reply:
These are SWAGs, based on what I've seen for RHOBS. But they can be overridden, I think in the end we'd keep these values pinned in some CR